examples/energy-management-app/esp32/main/main.cpp

```diff
@@ -74,9 +75,11 @@ using namespace ::chip::Credentials;
 using namespace ::chip::DeviceManager;
 using namespace ::chip::DeviceLayer;
 
-static EnergyEvseDelegate * gDelegate       = nullptr;
-static EnergyEvseManager * gInstance        = nullptr;
-static EVSEManufacturer * gEvseManufacturer = nullptr;
+static EnergyEvseDelegate * gEvseDelegate            = nullptr;
```

Comment:
Small point - but maybe worth doing - the folks who reviewed my PR's suggested changing to using:

static std::unique_ptr<EnergyEvseDelegate> gEvseDelegate;
static std::unique_ptr<EnergyEvseManager> gEvseInstance;
static std::unique_ptr<DeviceEnergyManagementDelegate> gDEMDelegate;
static std::unique_ptr<DeviceEnergyManagementManager> gDEMInstance;
static std::unique_ptr<EVSEManufacturer> gEvseManufacturer;

Reply:
Done